### ISSUE TITLE
Kernal Panic on Shutdown after Hat install

### ISSUE THREAD

**Issue by sigpaw** [pimoroni/picade-hat#46]
![pwrdn_panic](https://github.com/pimoroni/picade-hat/assets/18584547/3fe741b5-a9f2-49f0-9c3e-d2676cdedb8b)
[config.txt](https://github.com/pimoroni/picade-hat/files/14999604/config.txt)

Hello,

When pressing the power button on my Picade, the system enters the shutdown sequence. All looks well at first. Then it comes to the console message "reboot: Power down". It pauses for a few seconds and then displays the "Panic Message".

I'm running a new install of the 4.8 image obtained using the Raspberry Pi Imager. The problem was there immediately after first install. I've tried a fresh install a couple of times with the same results. I've also since performed a sudo apt-get update and sudo apt-get upgrade (perhaps around Apr 13th 2024) on my most recent fresh install with no improvement.

Once shutting down this way, pressing the power button a second time is met with no response. If I unplug my Pi and plug it back in, it starts up properly and runs RetroPi, EmulationStation, etc. --- Until the next Power Button press.

Lather, rinse, repeat. ;-)

BTW, I'm running a 4GB Raspi 4B with Retro-Pi 4.8 on top of Buster 5.10.103-v71+

Thanks for your help!
'Sig

**Comment by Gadgetoid** [comment]
I don't think this is directly related to Picade, since we're just gluing together device-tree overlays in a very standard way. It could potentially be an issue with "gpio-poweroff" but then I'd expect it to be more widespread.

This Pi forum thread suggests it might be SD card/power related - https://forums.raspberrypi.com/viewtopic.php?t=5647

Otherwise... I haven't seen this crop up before and don't have a clue :grimacing:  - I'll ping our support staff just in case they've seen it.

**Comment by liambeeton** [comment]
@sigpaw Please reach out to me, I would like to chat to you about your GItHub username. This is the only way to message you I could find.

**Comment by sigpaw** [comment]
Helo?

________________________________
